src/Makefile

```diff
@@ -44,9 +44,9 @@ RUNTIME_C_SRCS := \
 	dlload sys init task array dump staticdata toplevel jl_uv datatype \
 	simplevector runtime_intrinsics precompile \
 	threading partr stackwalk gc gc-debug gc-pages gc-stacks method \
-	jlapi signal-handling safepoint jloptions timing subtype rtutils \
+	jlapi signal-handling safepoint jloptions timing subtype \
```

Comment:
Does this mean this file doesn't get analyzed now?

Reply:
yes, see commit message. jl_printf is now a safepoint, so that file needs some refactoring that I thought would be too extensive for this commit.

Reply:
Why is jl_printf now a safepoint?

Reply:
The mutex around uv_write can cause gc.